clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -258,8 +258,13 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
   }, [cookie, options.debug]);
 
   const handleDismiss = useCallback(() => {
-    handleUpdatePreferences(ConsentMethod.DISMISS, initialEnabledNoticeKeys());
-  }, [handleUpdatePreferences, initialEnabledNoticeKeys]);
+    if (!consentCookieObjHasSomeConsentSet(cookie.consent)) {
```

Comment:
this addressed a bug where if someone returns to the page, loads the modal from the link, and then closes the modal, their previous consent was being wiped out!